Gifski/MainWindowController.swift

```diff
@@ -320,8 +337,38 @@ extension MainWindowController: NSMenuItemValidation {
 		switch menuItem.action {
 		case #selector(open)?:
 			return !isRunning
+		case #selector(quickLook(_:))?:
+			return conversionCompleted
 		default:
-			return validateMenuItem(menuItem)
```

Comment:
this was causing infinity loop as we called the same function we are in, and thus crash

Reply:
Good catch. More info here: https://stackoverflow.com/questions/49388800/falling-through-to-super-validatemenuitem-generates-swift-exception?rq=1